File: Programs/Gradebook/Python/Grades.py
class Grades(object):
    def __init__(self, student, course, exercises, exams, finals, finalGrade):
        self.student = student
        self.course = course
        self.exercises = exercises
        self.exams = exams
        self.finals = finals
        self.finalGrade = finalGrade

    def __repr__(self):
        return repr((self.student, self.course, self.exercises, self.exams, self.finals, self.finalGrade))

    def getStudent(self):
        return self.student

    def getCourse(self):
        return self.course

    def getExercises(self):
        return self.exercises

    def getExams(self):
        return self.exams

    def getFinals(self):
        return self.finals

    def getFinalGrade(self):
        return self.finalGrade

    def setStudent(self, student):
        self.student = student

    def setCourse(self, course):
        self.course = course

    def setExercises(self, exercises):
        self.exercises = exercises

    def setExams(self, exams):
        self.exams = exams

    def setFinals(self, finals):
        self.finals = finals

    def setFinalGrade(self, finalGrade):
        self.finalGrade = finalGrade

File: Programs/Gradebook/Scala/Grades.scala
class Grades(val newStudent: Int, val newCourse: Int, val newExercises: Float, val newExams: Float, val newFinals: Float, val newFinalGrade: Float){
	var student: Int = newStudent
	var course: Int = newCourse
	var exercises: Float = newExercises
	var exams: Float = newExams
	var finals: Float = newFinals
	var finalGrade: Float = newFinalGrade

	def getStudent(): Int = {
		return student
	}

	def getCourse(): Int = {
		return course
	}

	def getExercises(): Float = {
		return exercises
	}

	def getExams(): Float = {
		return exams
	}

	def getFinals(): Float = {
		return finals
	}

	def getFinalGrade(): Float = {
		return finalGrade
	}

	def setStudent(newStudent: Int) {
		student = newStudent
	}

	def setCourse(newCourse: Int) {
		course = newCourse
	}

	def setExercises(newExercises: Float) {
		exercises = newExercises
	}

	def setExams(newExams: Float) {
		exams = newExams
	}

	def setFinals(newFinals: Float) {
		finals = newFinals
	}

	def setFinalGrade(newFinalGrade: Float) {
		finalGrade = newFinalGrade
	}
}